src/Module/Link/Domain/Validation/RuleFactory.php

```diff
@@ -0,0 +1,24 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Module\Link\Domain\Validation;
+
+use Yiisoft\Validator\Rule\HasLength;
+use Yiisoft\Validator\Rule\Url;
+
+final class RuleFactory
```

Comment:
Can be done as in https://github.com/yiisoft/validator#grouping-common-validation-rules-into-rule-sets

Reply:
done